README.md

```diff
@@ -1,15 +1,64 @@
 # Flair HQ
 
-## Setup
-
-1. Make sure you have Node (>=4.0.0), and MongoDB installed
-1. Clone the repository `git clone https://github.com/YaManicKill/flairhq.git`
-1. Navigate into the directory `flairhq`
-1. Run `npm install`  to install the dependencies
-1. Copy config/local.example.js to config/local.js
-1. Create a reddit app on [https://www.reddit.com/prefs/apps](https://www.reddit.com/prefs/apps)
-1. Copy the id and secret to config/local.js
-1. Use something like [this](https://github.com/not-an-aardvark/reddit-oauth-helper) to get a refresh token for a moderator on the subs
-1. Start your MongoDB
-1. Start sails with `npm start`
-1. Open `http://localhost:1337` in your browser
+## How to setup your own dev environment
+
+### 1. Create two subreddits for testing
+
+You probably don't to do testing on production so the we suggest to create two subreddits for testing purposes. [Start here](https://www.reddit.com/subreddits/create). You can choose any names for it (such as yourUsername and yourUsername1). Once you have the subreddits created, create another reddit account.  
+Your first account will automatically get mod permission on both subreddits and the other one you can use to testing from end user point of view.
+
+### 2. Install prerequisites
+
+   Make sure you have Node.js ([some helpful guides here](https://nodejs.org/en/download/package-manager/)) and MongoDB ([check it here](https://docs.mongodb.com/manual/installation/)) installed (you can install both of them using default settings). If possible install the newest stable release. For Node.js version 4.0.0 is minimal.
```

Comment:
"Node.js 4.0.0 or greater is required"

Remove "(you can install both of them using default settings)" and "If possible install the newest stable release". Unnecessary sentences and you don't want too much text because people won't read it.

Reply:
Maybe add git to this section rather than as a sidenote in the next.